package/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,118 @@
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+   * Threats or violent language directed against another person
+   * Discriminatory jokes and language
+   * Posting sexually explicit or violent material
+   * Posting (or threatening to post) other people's personally identifying
+     information ("doxing")
+   * Personal insults, especially those using racist or sexist terms
+   * Unwelcome sexual attention
+   * Advocating for, or encouraging, any of the above behavior
+   * Repeated harassment of others. In general, if someone asks you to stop,
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time.
+   They are probably not purposefully ignoring issues, refusing to engage in
+   discussion, avoiding features, etc. but often just unavailable.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and MDAnalysis is no exception. It is important
+   that we resolve disagreements and differing views constructively. Remember
+   that we’re different. The strength of MDAnalysis comes from its varied community
+   that includes people from a wide range of backgrounds. Different people have
+   different perspectives on issues. Being unable to understand why someone
+   holds a viewpoint doesn’t mean they’re wrong. Don’t forget that it is human
+   to err and blaming each other doesn’t get us anywhere. Instead, focus on
+   helping to resolve issues and learning from mistakes.
+
+8. **A simple apology can go a long way**. It can often de-escalate a situation,
+   and telling someone that you are sorry is act of empathy that doesn’t
+   automatically imply an admission of guilt (Canada has turned
+   [*this notion into law*](http://www.theloop.ca/canadians-love-to-say-sorry-so-much-we-had-to-make-this-law/)).
+
+
+## Reporting
+
+If someone makes you or any other contributor feel unsafe or unwelcome,
+please report this in a timely manner. Code of conduct violations reduce
+the value of the community for everyone and we take them seriously.
+
+You can file a report by emailing
+[*special-list?*](TODO) or by filing out
+[this form](TODO). For more details or
+information on reporting in-person at an event, please see our Reporting
+Guidelines.
+
+The online form gives you the option to keep your report anonymous or request
```

Comment:
I also have no idea. I would be OK though if we add a non-public mailing list that people can use to write emails to. I would like as least one non-public option. That is important for me. The form might be nice since it lowers the barrier. 

Here is the jupyter [form](https://docs.google.com/forms/d/e/1FAIpQLSfUuaGbXrH0anroNIkbc-dBrn-ISsC_BA57IRt57aQW85nK6w/viewform?c=0&w=1)